hw/ip/lc_ctrl/data/lc_ctrl_sec_cm_testplan.hjson

```diff
@@ -155,7 +155,7 @@
       into `IdleSt`).
       '''
```

Comment:
can you also add this is verified by the FPV test - lc_ctrl_sec_cm_fsm?
Current dvsim doesn't allow to add a FPV test for a testpoint, but we can at least add it in the description.

Reply:
Ok done.